repo/content/content_manager.go

```diff
@@ -40,6 +40,11 @@ const (
 	DefaultIndexVersion = 2
 )
 
+// ManifestContentPrefix is the prefix of the content id for manifests.
+const (
+	ManifestContentPrefix = "m"
```

Comment:
this is a layering inversion, let's not do it.

Reply:
@jkowalski just to clear my confusion, are you suggesting to remove complete handling for manifest content type?

Reply:
@jkowalski could you please help me understand this, are you suggesting to remove all the changes related to manifest content type?

Reply:
what I was saying, content manager should not have special cases for any particular content types. Manifest contents should only be manipulated by "manifest" package.